libc/test/src/__support/arg_list_test.cpp

```diff
@@ -10,6 +10,50 @@
 
 #include "test/UnitTest/Test.h"
 
+#include "src/__support/OSUtil/io.h"
+#include "src/__support/integer_to_string.h"
+using namespace LIBC_NAMESPACE;
+
+namespace {
+
+void nl() { write_to_stderr("\n"); }
+void dump(const char *s) {
+  write_to_stderr(s);
+  nl();
+}
+
+void dump(int32_t s) {
```

Comment:
Probably it's better to do:
```
template <typename T>
cpp::enable_if_t<cpp::is_integral_v<T>, void> dump(T s) {
  auto x = IntegerToString<T>(s);
  write_to_stderr(x.view());
  nl();
}
```
Then all the usages below could be changed to `dump(s.*);`

Reply:
Ah rubbish, the libc commit isn't supposed to be in this PR. That's a collection of local hacks trying to get libc to build and tell me useful things, which I've uploaded in error while patching Matt's comments. I'll work out how to remove that

Reply:
Now removed